src/pack.c

```diff
@@ -965,62 +993,75 @@ int get_delta_base(
  *
  ***********************************************************/
 
-void git_packfile_close(struct git_pack_file *p, bool unlink_packfile)
+/* Run with the mwindow lock held */
+void git_packfile_close_locked(struct git_pack_file *p, bool unlink_packfile)
 {
+	bool locked = true;
+	if (git_mutex_lock(&p->lock) < 0) {
+		git_error_set(GIT_ERROR_OS, "failed to lock pack file");
```

Comment:
let's be consistent with "packfile"
```suggestion
		git_error_set(GIT_ERROR_OS, "failed to lock packfile");
```

Reply:
huh, i have no idea where i got this from.